### PR TITLE
Adding a logomark/logotype

### PR DESCRIPTION
**Pull request by ramiromagno**
New logo proposal. I've used an AI-generated simple watercolor of an oak branch and leaves, and made the leaves' hue inspired on CDISC's logomark colors, namely the colored dots. The dark blue of the outline and of the text is the same of CDISC's main dark blue. I think it looks nice and simple. Let me know your thoughts!

sdtm oak-wo-fonts

### CODE REVIEW

**Review by kamilsi — DISMISSED** [review]
Hi @ramiromagno,

I absolutely love the new logo proposal! The AI-generated watercolor of an oak branch and leaves is stunning, and the inspiration drawn from CDISC's logomark colors for the leaves adds a beautiful touch of continuity and homage. The choice of dark blue for the outline and text, matching CDISC's main color, ties everything together perfectly. It's both nice and simple, exactly as you intended.

I have only one suggestion, aimed at fostering future adaptability. Could we include the R code that generated this image in the repository? I understand the hex might have been created using a project similar to [hexSticker](https://cran.r-project.org/web/packages/hexSticker/index.html). Having the code accessible would allow us to modify elements like the outline color easily if we decide to make changes down the line.

**Comment by ramiromagno** [comment]
Hi Kamil,

Thank you for your nice comments about the logo!

I haven't used any R code other than `usethis::use_logo()` to place the logo in the right place. I have a long time ago developed my hex border with the right dimensions that I reuse for any new logo creation. So there isn't any special trick other than opening it in Inkscape and editing the logo to your liking and saving it. Then `usethis::use_logo()` takes the SVG file and places it where it needs to be.

So, if you want, you can just take this logo (see this PR's man/figures/logo.svg), open it in Inkscape and adapt it to your needs.

**Comment by rammprasad** [comment]
Hi @ramiro, Looks great. Can we make it green to match the OAK theme? Also, a simpler leaf will be better. I kind of managed to create this, but couldn't get the sdtm.oak inside the hexagon.

My first design

**Comment by ramiromagno** [comment]
Do you mean something like this?

sdtm oak_ramm

(I still prefer my own version :) )

**Comment by rammprasad** [comment]
Exactly, this is what I meant. We can even replace the oak leaf with an oak tree-like thing.
Designer

**Comment by ramiromagno** [comment]
Absolutely! After all this is open source :)

**Comment by ramiromagno** [comment]
As discussed in today's meeting I will merge the new logo into main.

File: README.Rmd
Comment (edgar-manukyan) [inline README.Rmd:1]: @rammprasad, is the statement "data in ODM format" still true?

**Review by edgar-manukyan — APPROVED** [review]
Thanks so much Ramiro, looks awesome! ❤️

File: README.Rmd
Comment (rammprasad) [reply README.Rmd:1]: No, it is not. Where is it specified? We can remove it.

**Review by rammprasad — COMMENTED** [review]
Added my comments.

File: README.Rmd [hunk 1]
```diff
@@ -13,7 +13,7 @@ knitr::opts_chunk$set(
 )
 ```
 
-# sdtm.oak
+# sdtm.oak <a href="https://pharmaverse.github.io/sdtm.oak"><img src="man/figures/logo.svg" align="right" height="139" /></a>
 
 <!-- badges: start -->
 [![CRAN status](https://www.r-pkg.org/badges/version/sdtm.oak)](https://CRAN.R-project.org/package=sdtm.oak)
```
[the remark this answers is not in the document]
Comment (ramiromagno) [reply README.Rmd:19]: Done.

**Comment by ramiromagno** [comment]
@rammprasad : I think all requests have been addressed and we can merge this.

**Review by rammprasad — APPROVED** [review]
Great job.

**Pull request merged**